### ISSUE TITLE
Style for images and 3D (three.js)

### ISSUE THREAD

**Issue by AlanRynne** [Paramdigma/paramdigma.github.io#6]
Should we even bother with 2D images?

If so, generate a style for them in Illustrator/Inkscape or find a not manual solution.

3D solution might fix both problems, as Three.js also renders images and vector graphics, this might be a harder but better way to go (plus it will serve as the base for a super cool tutorial)

What do you think @christiandimitri?

**Comment by AlanRynne** [comment]
This is a reference of the possible appearance in 3D

https://threejs.org/examples/#webgl_materials_variations_toon

**Comment by christiandimitri** [comment]
Yeah I find this super interesting. Well for 2D I don't find other alternative than implementing the 2D algorithms explained (ex: tree, growth,...) with sliders of the parameters on the webpage.
